### PR TITLE
Official Website banner: simplified / Global header changes

### PR DESCRIPTION
**Pull request by ericronne**
[Click and behold.](https://federalist.18f.gov/preview/18F/doi-extractives-data/map-color-nothingclever/)

NOTE: branch name is incorrect (in addition to being unclever). No map changes were made.

### CODE REVIEW

**Comment by ericronne** [comment]
Changes were all to the banner:
 - Removed the feedback and what's new links from the banner. They appeared to be suffering from "banner blindness," and thus received very few clicks. "What's new" is a high-profile link atop the About main page, and the feedback link is still available via a large button in the footer.
- Centered the Official notice and made it slightly larger, bold
- Slab color change

File: _sass/blocks/_header.scss [hunk 1]
```diff
@@ -53,6 +52,10 @@ header,
   @include respond-to(medium-up) {
     @include span-columns(3);
   }
+
+  @include respond-to(medium-down) {
+    margin-bottom: $base-padding;
```
Comment (gemfarmer) [inline _sass/blocks/_header.scss]: @ericronne you could probably do something like this instead:

```
.header-left {
   @include span-columns(6);
   margin-bottom: $base-padding;

   @include respond-to(medium-up) {
      @include span-columns(3);
      margin-bottom: 0;
   }
}

File: _sass/blocks/_header.scss [hunk 2]
```diff
@@ -106,16 +109,17 @@ header,
   margin-bottom: 0;
 
   @include respond-to(huge-plus-up) {
-    margin-bottom: 1em;
+    margin-bottom: 2em;
```
Comment (gemfarmer) [inline _sass/blocks/_header.scss]: Would it be possible to use `$base-padding` instead of `2em`? That is `1.25em`

Comment (gemfarmer) [inline _sass/blocks/_header.scss]: Trying to stick to variables as much as possible

**Comment by gemfarmer** [comment]
@ericronne It appears that the banner disappears in mobile ☹️ 
screen shot 2017-03-29 at 4 53 16 pm

**Comment by ericronne** [comment]
The gov bit vanishes on the live site, too …
image

I'll see if i can fix; it should be there.

**Pull request merged**